keps/sig-node/runtime-class-scheduling.md

```diff
@@ -138,13 +138,12 @@ The RuntimeClass definition is augmented with an optional `Topology` struct:
 
 ```go
 type Topology struct {
-    // nodeSelectorTerm selects the set of nodes that support this RuntimeClass.
-    // Pods using this RuntimeClass can only be scheduled to a node matched by this selector.
-    // The nodeSelectorTerm is merged with a pod's other node affinity match
-    // expressions by appending the additional requirements to each preexisting
-    // NodeSelectorTerm.
+    // nodeSelector selects the set of nodes that support this RuntimeClass.
+    // Pods using this RuntimeClass can only be scheduled to a node matched by
+    // this selector. The nodeSelector is intersected (AND) with a pod's other
+    // node affinity or node selector requirements.
     // +optional
-    NodeSelectorTerm []corev1.NodeSelectorRequirement
+    NodeSelector corev1.NodeSelector
```

Comment:
Is this change due to the preference of adopting a new scheduler predicate, so that we don't need to mix-in this info?

Reply:
Yes. If we don't need to worry about logically merging this with the existing pod's node affinity, then I prefer the more expressive `NodeSelector` over a `NodeSelectorTerm`. I could be convinced otherwise though.

Reply:
not really opposed this.